src/Employer/Employer.Web/Orchestrators/LevyDeclarationOrchestrator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using Esfa.Recruit.Employer.Web.Configuration.Routing;
+using Esfa.Recruit.Employer.Web.Extensions;
+using Esfa.Recruit.Employer.Web.ViewModels.LevyDeclaration;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+
+namespace Esfa.Recruit.Employer.Web.Orchestrators
+{
+    public class LevyDeclarationOrchestrator
+    {
+        private readonly IEmployerVacancyClient _client;
+
+        public LevyDeclarationOrchestrator(IEmployerVacancyClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<LevySelectionOrchestratorResponse> SaveSelectionAsync(LevyDeclarationModel viewModel, ClaimsPrincipal user)
+        {
+            if (viewModel.ConfirmAsLevyPayer.Value)
+                await _client.SaveLevyDeclarationAsync(user.GetUserId(), viewModel.EmployerAccountId);
+
+            return new LevySelectionOrchestratorResponse 
+            {
+                RedirectRouteName = viewModel.ConfirmAsLevyPayer.Value ? RouteNames.Dashboard_Index_Get : RouteNames.NonLevyInfo_Get,
+                CreateLevyCookie = viewModel.ConfirmAsLevyPayer.Value
+            };
+        }
+    }
+
+    public class LevySelectionOrchestratorResponse
```

Comment:
Could we not use or extend off the Orchestrator response class?

Reply:
I'm not sure it sits very well Orchestrator Response class as that class is more focussed towards that validation of entity operation. Whereas what I need here is the choice of where to navigate the user to.